infra/kubernetes/main.tf

```diff
@@ -0,0 +1,81 @@
+// terraform config //
+terraform {
+  required_providers {
+    aws = {
+      source  = "hashicorp/aws"
+      version = "~> 3.0"
+    }
+  }
+  # backend "s3" {
+  #   bucket  = "cncf-credits-infra-kubernetes-tfstate"
+  #   key     = "terraform.tfstate"
+  #   region  = "ap-southeast-2"
+  #   encrypt = true
+  # }
+}
+
+provider "aws" {
+  profile = "default"
+  region  = "ap-southeast-2"
+}
+// ---------------- //
+
+// the tfstate for the management of the kubernetes account+OU //
+resource "aws_kms_key" "cncf-credits-infra-kubernetes-tfstate" {
```

Comment:
Can be dropped. it's not needed at the moment.

Reply:
See https://github.com/cncf/credits/pull/3#discussion_r796181359